pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -38,8 +38,8 @@
 using pwiz.Skyline.Model.Irt;
 using pwiz.Skyline.Model.Results;
 using pwiz.Skyline.Model.Results.Scoring;
-using pwiz.Skyline.Model.Tools;
 using pwiz.Skyline.Properties;
+using pwiz.Skyline.ToolsUI;
```

Comment:
This should be enough to tell you, these changes have a problem. You are not allowed to include a UI namespace in CommandArgs.